src/box/raft.c

```diff
@@ -55,6 +55,9 @@ static struct trigger box_raft_on_update;
 struct rlist box_raft_on_broadcast =
 	RLIST_HEAD_INITIALIZER(box_raft_on_broadcast);
 
+struct rlist box_raft_on_message_send_cb =
```

Comment:
It seems you don't need triggers. You would be fine with a simple `fiber_cond`. Because the only user of this trigger is a blocking wait. `box_raft_wait_term_broadcasted` would wait on the cond. This cond could be called `box_raft_on_message_cond`.